src/Data/Repositories/Interfaces/IWalletRepository.cs

```diff
@@ -35,8 +35,9 @@ public interface IWalletRepository
     /// <summary>
     /// Obtains all wallets that are Finalised and not in a compromised or archived state
     /// </summary>
+    /// // <param name="includeWatchOnlyWallets">If true, only wallets with an internal wallet will be returned</param>
     /// <returns> List of available wallets</returns>
-    Task<List<Wallet>> GetAvailableWallets();
+    Task<List<Wallet>> GetAvailableWallets(bool includeWatchOnlyWallets = true);
```

Comment:
why are you declaring it in the interface as a true by default and then in the respository as false by default?

Reply:
Typo, ill fix it

Reply:
Fixed